plugins/modules/aci_rest.py

```diff
@@ -369,6 +371,7 @@ def main():
         with open(src, 'r') as config_object:
             # TODO: Would be nice to template this, requires action-plugin
             payload = config_object.read()
+            pld = json.loads(payload.replace("\'", '"'))
```

Comment:
we should find a better name than pld which is not explanatory enough. Or you could move the call directly to where you need it to remove the need for pld. Also you can use the following example to remove the need for the replace: https://coderwall.com/p/hvqbpg/parse-json-with-single-quotes-in-python

Reply:
ok